pom.xml

```diff
@@ -1835,6 +1835,8 @@
                                 <exclude>.editorconfig</exclude>
                                 <exclude>**/hadoop.indexer.libs.version</exclude>
                                 <exclude>**/codegen/**</exclude>
+                                <exclude>website/target/**</exclude>
```

Comment:
IMHO These exclusions should not be required.

Reply:
@cryptoe This is not caused by my changes, but if that exclusion is not included, Travis will always fail during this maven task.

`target/` folder is always auto generated code, yes? Isn't it ok to exclude?

Reply:
As your py changes are not related to pom.xml, IMHO these changes are not required. 
If you think stuff is already broken in the master, please feel free to raise a separate PR.

Reply:
@cryptoe I can revert this.

Reply:
Reverted. Raised this instead: https://github.com/apache/druid/pull/12358